scripts/assets/file_content_validation_checks.textproto

```diff
@@ -306,3 +306,50 @@ file_content_checks {
   exempted_file_name: "utility/src/test/java/org/oppia/android/util/math/RealExtensionsTest.kt"
   exempted_file_patterns: "testing/src/main/java/org/oppia/android/testing/junit/.+?\\.kt"
 }
+
```

Comment:
Nit: please remove blank newline for consistency.

Reply:
Done